app/pb_kits/playbook/pb_date_year_stacked/docs/_date_year_stacked_default.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
```

Comment:
@thestephenmarshall Markie is creating a helper for dates on the rails side. What do we want to do on react? Moment.js?

Reply:
`moment` +1